pheweb/load/phenolist_gbma.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env/python3
```

Comment:
can you add comment for what this is used for.  I was wondering should we have some other place for these types of things. I don't have any strong opinion though.